src/bin/cql-stress-cassandra-stress/settings/cs_args_good_test.in

```diff
@@ -28,3 +28,5 @@ cassandra-stress read cl=QUORUM n=10000 -schema replication(key=value) -rate thr
 cassandra-stress read cl=QUORUM n=10000 -schema replication(factor=1, key=value) -rate threads=10 -col size=FIXED(50)
 cassandra-stress read cl=QUORUM n=10000 -schema replication(factor=1, key=value) -rate threads=10 -col size=FIXED(50) -pop dist=SEQ(1..10)
 cassandra-stress read cl=QUORUM n=10000 -schema replication(factor=1, key=value) -rate threads=10 -col size=FIXED(50) -pop dist=UNIFORM(1..10)
+cassandra-stress read cl=QUORUM n=10000 -schema replication(factor=1, key=value) -rate threads=10 -col size=fixed(50) -pop dist=UNIFORM(1..10)
+cassandra-stress read no-warmup cl=QUORUM duration=600M -rate threads=80 throttle=8000/S
```

Comment:
I suppose that `throttle=8000/S` is the parameter that checks that case-insensitivity works - because of the uppercase `S` - right? I think it's easy to miss that, maybe using e.g. `cl=QuOrUm` would make it more explicit instead.

How about putting a comment on the previous line which mentions that this test is about case insensitivity? The `cs_args_good_test` test case already ignores lines that start with a hash sign.